test/gleam/dynamic_test.gleam

```diff
@@ -1132,14 +1132,14 @@ pub fn dict_test() {
   |> dynamic.from
   |> dynamic.dict(dynamic.int, dynamic.int)
   |> should.equal(
-    Error([DecodeError(expected: "Int", found: "String", path: ["keys"])]),
+    Error([DecodeError(expected: "Int", found: "String", path: ["keys[0]"])]),
```

Comment:
This isn't the syntax for indexing in this module, and you can't index into the keys with `0`.

Reply:
Hi

Not sure what do you mean with the syntax comment.

The `values[1]` is just meant to show the index of the failed input. Changed the test to reflect this.

The intention is to have some way to find the problematic input in the original data source.

e.g. given
```json
{
  "dog: { "id": 1 },
  "cat": { "id": 2 },
  "fish": { "id": null },
  "rabbit": { "id": 4 },
}
```

I will like to be able to see an error showing `values[2]` which points me to `fish`.

Not sure however if the index will be correct when parsing something like this.

Reply:
The indexing only works on erlang. We need a different strategy.